python/ray/train/v2/tests/test_controller.py

```diff
@@ -173,28 +178,46 @@ def test_resize():
         ResizeDecision(num_workers=5, resources_per_worker={}),
     ]
 
+    assert isinstance(controller.get_state(), InitializingState)
+    worker_group_status = worker_group.poll_status()
+    assert worker_group_status.num_workers == 0
```

Comment:
maybe we should not allow  `poll_status` on a dead worker group?

Reply:
Yeah, I'm handling this in a subsequent PR that focuses on the worker group. Will think holistically about what it means to start/shutdown a WorkerGroup there.